City-of-Helsinki/drupal-hdbt-admin
# UHF-X Design selection fallback

How to test: 
- `make new`
- Checkout this branch and remove one or more images from `public/themes/contrib/hdbt_admin/modules/hdbt_admin_editorial/assets/images` but not the actual fallback image named `custom-style.svg`
   - Make a note of the images you remove and go to edit a landing page or basic page where the images might be used. F.e. `paragraph--columns.svg` is in use when previewing columns paragraph and `hero-design--diagonal.svg` is used when viewing hero style called diagonal.
- Check that the preview images get a fallback image when you try to hover a preview.
